besspin/cyberPhys/cyberphyslib/cyberphyslib/canlib/canlib.py

```diff
@@ -19,7 +19,7 @@ class CanConstants():
     DEST_BROADCAST = "<broadcast>"
     CAN_MIN_BYTES = 4 + 1 + 1  # sending an empty frame doesn't make sense, min 6 bytes per frame
     CAN_MAX_BYTES = 64 + 4 + 1  # 64 bytes of DATA, 4 bytes of ID, 1 byte od DLC
-    COMMAND_ID_PREFIX =  [0xAA, 0xFE, 0xEB] # CAN ID signifying a command message
+    COMMAND_ID_PREFIX =  [0xAA] # CAN ID signifying a command message
```

Comment:
I am in favor of removing this completely - it is not like a device is sending random data, so we *know* what is on the network. And this might bite us again in the future when we forget about the filtering feature.